internal/quickstart/messages.go

```diff
@@ -94,15 +94,15 @@ func sendCreateFlagMsg(client flags.Client, accessToken, baseUri, flagName, flag
 	}
 }
 
-type fetchedSDKInstructions struct {
+type fetchedSDKInstructionsMsg struct {
 	instructions []byte
 }
 
 type choseSDKMsg struct {
 	sdk sdkDetail
 }
 
-func sendChoseSDKMsg(sdk sdkDetail) tea.Cmd {
+func chooseSDK(sdk sdkDetail) tea.Cmd {
```

Comment:
Why did this change to present tense now?

Reply:
I'm trying to capture the intent of the command to represent something that should happen in a way we would naturally talk about it instead of forcing the naming around the name of the returned message.